vizro-core/src/vizro/static/css/button.css

```diff
@@ -8,41 +10,28 @@
   letter-spacing: var(--letter-spacing-body-link-02);
   line-height: var(--text-size-05);
   padding: var(--spacing-01) var(--spacing-03);
+  text-transform: none;
 }
 
-.button_primary:hover {
-  background: var(--fill-medium-emphasis);
-  text-decoration-line: underline;
-}
-
-.button_primary:focus {
-  box-shadow: 0 0 0 2px var(--focus-focus) inset;
+#page-container .button_primary:hover {
+  background: linear-gradient(
+      var(--state-overlays-contrast-hover),
+      var(--state-overlays-contrast-hover)
+    ),
+    var(--fill-active);
+  color: var(--text-contrast-primary);
+  margin: 0;
   text-decoration-line: underline;
 }
 
-.button_secondary {
-  background: var(--surfaces-bg-03);
-  border: 1px solid var(--border-enabled);
-  box-shadow: var(--box-shadow-elevation-0);
-  color: var(--text-primary);
-  font-size: var(--text-size-02);
-  font-weight: var(--text-weight-semibold);
-  height: 32px;
-  letter-spacing: var(--letter-spacing-body-link-02);
-  line-height: var(--text-size-05);
-  padding: var(--spacing-01) var(--spacing-03);
+#page-container .button_primary:active {
+  background: var(--fill-active);
+  color: var(--text-contrast-primary);
 }
 
-.button_tertiary {
```

Comment:
@nadijagraca - just FYI: I've removed the secondary and tertiary button because we didn't use it in our repository and I'd rather not have it in there then. Also I've removed the focus state. I had a quick catch-up with Pruthvi and he said that the design system will remove all focus states.